stable/wordpress/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: wordpress
-version: 0.8.18
+version: 0.8.19
```

Comment:
This adds a new feature, so you should bump the minor version. Didn't notice that on the one I lgtm'ed a bit ago.

Reply:
actually, it should be the major version as it is not backward compatible. I will also update the rest of PRs